config/packages/monolog.yaml

```diff
@@ -11,3 +13,40 @@ monolog:
       ident: stepup-selfservice
       facility: user
       formatter: surfnet_stepup.monolog.json_formatter
+
+when@dev:
+  monolog:
+    handlers:
+      prod-signaler:
+        action_level: ERROR
+        passthru_level: DEBUG # DEV setting: this means that all message of level DEBUG or higher are always logged
+        bubble: true
+      main_logfile:
+        type: stream
+        handler: logfile
+        level: NOTICE
+        path: "%kernel.logs_dir%/%kernel.environment%.log"
```

Comment:
If I have a say in this, I'd like to vote for logging everything to the stdout. And write nothing to var/logs.

Reply:
@MKodde Which project has a monolog.yaml we can reuse here?